livedata/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -203,9 +203,10 @@ class ChannelControllerImpl(
 
     fun sortedMessages(): List<Message> {
         // sorted ascending order, so the oldest messages are at the beginning of the list
-        val messageMap = _messages.value ?: mutableMapOf()
-        val messages = messageMap.values.sortedBy { it.createdAt }.filter { hideMessagesBefore == null || it.createdAt!! > hideMessagesBefore }
-        return messages
+        _messages.value?.let {
+            return it.values.sortedBy { it.createdAt }.filter { hideMessagesBefore == null || it.createdAt!! > hideMessagesBefore }
+        }
+        return emptyList()
```

Comment:
What do you think about to have only one return?
```suggestion
        return _messages.value?.let {
            it.values.sortedBy { it.createdAt }.filter { hideMessagesBefore == null || it.createdAt!! > hideMessagesBefore }
        }
        ?: emptyList()
```